src/gd_kotlin.cpp

```diff
@@ -64,6 +64,20 @@ void load_classes_hook(JNIEnv* p_env, jobject p_this, jobjectArray p_classes) {
     classes.delete_local_ref(env);
 }
 
+void register_src_dirs_hook(JNIEnv* p_env, jobject p_this, jobjectArray p_src_dirs) {
+    jni::Env env(p_env);
+    jni::JObjectArray src_dirs_raw{jni::JObjectArray(p_src_dirs)};
+    Vector<String> src_dirs{};
+    for (int i = 0; i < src_dirs_raw.length(env); ++i) {
+        String src_dir = env.from_jstring(src_dirs_raw.get(env, i));
+        src_dirs.push_back(src_dir);
+    }
+    GDKotlin::get_instance().scripts_root = src_dirs;
+    jni::JObject j_object{p_this};
+    j_object.delete_local_ref(env);
```

Comment:
IIRC it is a JVM to CPP call so there is no need to delete_local_ref.
Or am I missing something ?